### PR TITLE
[pulseaudio] For source, default timeout should be disconnection ASAP

### PR DESCRIPTION
**Pull request by dalgwen**
Hello all.
When reading from a pulseaudio audio source, a weird behaviour could happen.

Explanation:
There is a (configurable) grace timeout between disconnection. Each read from the socket pushes the timeout forward in time.
This is good for the sink part, as it reduces the need to reconnect socket (and so improve latency), but not so for the source part.

When we get for the first time an inputstream and pass it to the STT service, the binding connects to the socket and returns an appropriate inputstream whose data, hopefully, will immediately be consumed by a TTS service.
When all needed data is collected, the TTS closes the stream from its side (TTS choice, based on VAD or its own max timeout).
The socket is NOT closed. Because there is a pipe in between (usefull to send data to multiple input stream). It is the dedicated pipe that is closed.
The socket remains open during a grace period, allowing a new inputstream to consume data from it faster.
But as the socket remains open, with no inputstream consuming it, audio data is accumulating on the pulseaudio server side.
Then, if openHAB opens a new input stream (because a new vocal command is to be issued) on the socket during the grace period, the pulse audio server will send all accumulated data inbetween to it.

This bug can only happen if openHAB doesn't handle the wake word (because if so, the binding never stops to consume data).
It is the case with the new method "listenAndAnswer".

Solution: 
If the socket is disconnected immediately after use, no such behavior happens.
However, I would like to keep the graceperiod as a configuration parameter, for genericity with the sink part. And these weird behaviour could be usefull for whatever use case we cannot imagine right now.
So I suggest to make a new "0" timeout parameter that logically process to disconnect as soon as possible. And make it the default value for source, and let the parameter be an "advanced" parameter to suggest users to not meddle with it without really knowing what they do.

As the method "listenAndAnswer" is new, it could be seen as a regression introduced in 4.0.0  ? On the other side, I don't think many users will use/see this.
Actually, what is the policy for backporting bug fix in release patch branch ?

### CODE REVIEW

File: bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/i18n/pulseaudio.properties
Comment (GiviMAD) [inline bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/i18n/pulseaudio.properties:78]: I think we can put something more clarifying like "recommended value to avoid audio delay".

**Review by GiviMAD — CHANGES_REQUESTED** [review]
I have added a minor comment, otherwise LGTM.
Thank you for detecting the issue.

**Review by lolodomo — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/i18n/pulseaudio.properties
Comment (lolodomo) [reply bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/i18n/pulseaudio.properties:78]: Waiting for the feedback to this comment before merging.

Comment (dalgwen) [reply bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/i18n/pulseaudio.properties:78]: Indeed, it could be more clarifying. But it is not really an audio delay : it takes audio data from "before" the wanted data.
Maybe : "or expect irrevelant audio data capture"
(not satisfied by this either, I'm open to suggestion)

Comment (GiviMAD) [reply bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/i18n/pulseaudio.properties:78]: Maybe "to avoid problems with buffered audio data" or "to avoid capturing buffered audio"?

Comment (dalgwen) [reply bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/i18n/pulseaudio.properties]: Thank you, feels better.
"to avoid capturing unwanted buffered audio"
Done

**Comment by dalgwen** [comment]
Thank you @lolodomo

**Pull request merged**
